lib/inquisitor.ex

```diff
@@ -106,4 +113,16 @@ defmodule Inquisitor do
     |> List.last()
     |> Mix.Utils.underscore()
   end
+
+  @doc false
+  def handle_non_integer(i) when is_integer(i), do: i
+  def handle_non_integer(i) do
```

Comment:
I'm not certain that non-integers should be rescued.  It might make more sense to return a tuple with `{:error, reason}`.  Thoughts?